From 6c089c4f0c306e2192ec17f85c12c71f8ee87794 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sun, 15 Jan 2017 11:51:48 -0500
Subject: [PATCH] cryptsetup: fix unitialized variable

CID #1368416.

Coverity web interface is back, yay!

(cherry picked from commit 6fefc0eded00974e5b2862bd5efc7ef975c6a318)
---
 src/cryptsetup/cryptsetup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cryptsetup/cryptsetup.c b/src/cryptsetup/cryptsetup.c
index ff5a3f36fb..a90bee81bb 100644
--- a/src/cryptsetup/cryptsetup.c
+++ b/src/cryptsetup/cryptsetup.c
@@ -752,6 +752,7 @@ int main(int argc, char *argv[]) {
 
         } else {
                 log_error("Unknown verb %s.", argv[1]);
+                r = -EINVAL;
                 goto finish;
         }
 
-- 
2.9.3

